cmd/commands.go

```diff
@@ -305,6 +305,54 @@ the KEY=VALUE format will be loaded into the Caddy process.
 		},
 	})
 
+	RegisterCommand(Command{
+		Name:  "storage",
+		Short: "Commands for working with Caddy's storage",
+		Long: `
+Allows exporting and importing Caddy's storage contents. The two commands can be
+combined in a pipeline to transfer directly from one storage to another:
+
+$ caddy storage export --config Caddyfile.old --output - |
+> caddy storage import --config Caddyfile.new --input -
+
+With - referring to stdout/stdin.
+
+NOTE: When importing to or exporting from file_system storage (the default), the command
+should be run as the user that owns the associated root path.
+`,
+		CobraFunc: func(cmd *cobra.Command) {
+			exportCmd := &cobra.Command{
+				Use:   "export --config <path> --output <path>",
+				Short: "Exports storage assets as a tarball",
+				Long: `
+The contents of the configured storage module (TLS certificates, autosave.json, etc) 
```

Comment:
I'm pretty sure autosave.json goes into the config directory (`~/.config/caddy` default on Linux), not storage :thinking:

Reply:
Correct, config storage is not the same as data storage. Explained here https://caddyserver.com/docs/conventions#file-locations

Reply:
Thanks! I thought I saw it in exports at one point, but I'm sure I just did something weird myself.